### PR TITLE
lightning: refine description about checksum-via-sql

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ pd-addr = "172.16.31.4:2379"
 # 设置 TiDB 库的日志等级。
 log-level = "error"
 
-# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。
+# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。请注意，如果将 checksum-via-sql 参数设置为 true，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数 distsql-scan-concurrency = 15 和 checksum-table-concurrency = 2 将不会生效。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:306]:
```suggestion
# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 参数设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，参数 distsql-scan-concurrency = 15 和 checksum-table-concurrency = 2 将不会生效。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -345,6 +345,8 @@ max-allowed-packet = 67_108_864
 # - "optional"。在导入完成后执行 CHECKSUM 检查，如果报错，会输出一条 WARN 日志并忽略错误。
 # - "off"。导入结束后不执行 CHECKSUM 检查。
 # 默认值为 "required"。从 v4.0.8 开始，checksum 的默认值由此前的 "true" 改为 "required"。
+# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作，默认值为 false，表示通过 Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。建议将该值设为 true，以便在 checksum 失败时更容易定位问题。同时，当该值为 true 时，如果需要调整并发，请在 TiDB 中设置 tidb_checksum_table_concurrency 参数。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:348]:
```suggestion
# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作，默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置 tidb_checksum_table_concurrency 参数。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -303,7 +303,7 @@ pd-addr = "172.16.31.4:2379"
 # 设置 TiDB 库的日志等级。
 log-level = "error"
 
-# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。
+# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数 distsql-scan-concurrency = 15 和 checksum-table-concurrency = 2 将不会生效。
 # 各参数定义可参阅”控制 Analyze 并发度“文档
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:307]:
```suggestion
# 各参数定义可参阅 “控制 Analyze 并发度”文档 https://docs.pingcap.com/zh/tidb/stable/statistics#%E6%8E%A7%E5%88%B6-analyze-%E5%B9%B6%E5%8F%91%E5%BA%A6
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -303,8 +303,8 @@ pd-addr = "172.16.31.4:2379"
 # 设置 TiDB 库的日志等级。
 log-level = "error"
 
-# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。
-# 各参数定义可参阅”控制 Analyze 并发度“文档
+# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数 distsql-scan-concurrency = 15 和 checksum-table-concurrency = 2 将不会生效。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:306]:
```suggestion
# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数 `distsql-scan-concurrency = 15` 和 `checksum-table-concurrency = 2` 将不会生效。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -345,6 +345,8 @@ max-allowed-packet = 67_108_864
 # - "optional"。在导入完成后执行 CHECKSUM 检查，如果报错，会输出一条 WARN 日志并忽略错误。
 # - "off"。导入结束后不执行 CHECKSUM 检查。
 # 默认值为 "required"。从 v4.0.8 开始，checksum 的默认值由此前的 "true" 改为 "required"。
+# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作，默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置 tidb_checksum_table_concurrency 参数。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:348]:
```suggestion
# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作，默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置 `tidb_checksum_table_concurrency` 参数。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 6]
```diff
@@ -345,6 +345,8 @@ max-allowed-packet = 67_108_864
 # - "optional"。在导入完成后执行 CHECKSUM 检查，如果报错，会输出一条 WARN 日志并忽略错误。
 # - "off"。导入结束后不执行 CHECKSUM 检查。
 # 默认值为 "required"。从 v4.0.8 开始，checksum 的默认值由此前的 "true" 改为 "required"。
+# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作，默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置 `tidb_checksum_table_concurrency` 参数。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:348]:
```suggestion
# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作。
# 默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。
# 建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。
# 同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置 `tidb_checksum_table_concurrency` 参数。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 7]
```diff
@@ -350,6 +350,11 @@ max-allowed-packet = 67_108_864
 # 1. Checksum 对比失败通常表示导入异常（数据丢失或数据不一致），因此建议总是开启 Checksum。
 # 2. 考虑到与旧版本的兼容性，依然可以在本配置项设置 `true` 和 `false` 两个布尔值，其效果与 `required` 和 `off` 相同。
 checksum = "required"
+# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作。
+# 默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。
+# 建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。
+# 同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置。`tidb_checksum_table_concurrency` 参数。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:356]:
```suggestion
# 同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置。`tidb_checksum_table_concurrency` 参数 (https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency)。
```

**Review by hfxsd — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 8]
```diff
@@ -303,8 +303,8 @@ pd-addr = "172.16.31.4:2379"
 # 设置 TiDB 库的日志等级。
 log-level = "error"
 
-# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。
-# 各参数定义可参阅”控制 Analyze 并发度“文档
+# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数 `distsql-scan-concurrency = 15` 和 `checksum-table-concurrency = 2` 将不会生效。
+# 各参数定义可参阅 “控制 Analyze 并发度”文档 https://docs.pingcap.com/zh/tidb/stable/statistics#%E6%8E%A7%E5%88%B6-analyze-%E5%B9%B6%E5%8F%91%E5%BA%A6
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:307]:
```suggestion
# 各参数定义可参阅 “控制 Analyze 并发度文档” (https://docs.pingcap.com/zh/tidb/stable/statistics#%E6%8E%A7%E5%88%B6-analyze-%E5%B9%B6%E5%8F%91%E5%BA%A6)。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 9]
```diff
@@ -350,6 +350,11 @@ max-allowed-packet = 67_108_864
 # 1. Checksum 对比失败通常表示导入异常（数据丢失或数据不一致），因此建议总是开启 Checksum。
 # 2. 考虑到与旧版本的兼容性，依然可以在本配置项设置 `true` 和 `false` 两个布尔值，其效果与 `required` 和 `off` 相同。
 checksum = "required"
+# 设置是否通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> 操作。
+# 默认值为 "false"，表示通过 TiDB Lightning 下发 ADMIN CHECKSUM TABLE <table> 命令给 TiKV 执行。
+# 建议将该值设为 "true"，以便在 checksum 失败时更容易定位问题。
+# 同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置。`tidb_checksum_table_concurrency` 参数 (https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency)。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:356]:
```suggestion
# 同时，当该值为 "true" 时，如果需要调整并发，请在 TiDB 中设置 `tidb_checksum_table_concurrency` 变量 (https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency)。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 10]
```diff
@@ -303,8 +303,8 @@ pd-addr = "172.16.31.4:2379"
 # 设置 TiDB 库的日志等级。
 log-level = "error"
 
-# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。
-# 各参数定义可参阅”控制 Analyze 并发度“文档
+# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数 `distsql-scan-concurrency = 15` 和 `checksum-table-concurrency = 2` 将不会生效。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:306]:
```suggestion
# 设置 TiDB 会话变量，提升 Checksum 和 Analyze 的速度。注意，如果将 checksum-via-sql 设置为 "true"，则会通过 TiDB 执行 ADMIN CHECKSUM TABLE <table> SQL 语句来进行 Checksum 操作。在这种情况下，以下参数设置 `distsql-scan-concurrency = 15` 和 `checksum-table-concurrency = 2` 将不会生效。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**